draft-ietf-ice-pac.xml

```diff
@@ -129,7 +129,7 @@
         </t>
         <section title="No Candidates From Peer">
           <t>
-            Per RFC XXXX, an ICE agent could provide zero candidates of
+            Per RFC XXXX, an ICE agent might provide zero candidates of
```

Comment:
I still think 'may' is the ideal word here. 'can' could also work.

Reply:
Ok, I will use "can".